owncloudTestUtil/src/main/java/com/owncloud/android/testutil/OCAppRegistryMimeType.kt

```diff
@@ -0,0 +1,13 @@
+package com.owncloud.android.testutil
+
+import com.owncloud.android.domain.appregistry.model.AppRegistryMimeType
+
+val OC_APP_REGISTRY_MIMETYPE = AppRegistryMimeType(
+    mimeType = "DIR",
+    ext = "appRegistryMimeTypes.ext",
+    appProviders = emptyList(),
+    name = "appRegistryMimeTypes.name",
+    icon = "appRegistryMimeTypes.icon",
+    description= "appRegistryMimeTypes.description",
+    allowCreation=true,
+    defaultApplication= "appRegistryMimeTypes.defaultApplication")
```

Comment:
A comma is missing here. And put the ) in the next line.

Reply:
You forgot to put the ')' in the next line.

Reply:
Done